setup-homebrew/main.sh

```diff
@@ -16,16 +16,16 @@ ohai() {
     echo -e "\\033[34m==>\\033[0m \\033[1m$*\\033[0m"
 }
 
-MAX_GIT_RETRIES=5
+MAX_RETRIES=10
 
-function git_retry {
+function retry {
     local try=0
 
-    until git "$@"; do
+    until "$@"; do
         exit_code="$?"
         try=$((try + 1))
 
-        if [ $try -lt $MAX_GIT_RETRIES ]; then
+        if [ $try -lt $MAX_RETRIES ]; then
             sleep $((2 ** try))
```

Comment:
I wonder if `4 ** (try + 1)` or similar would make more sense than bumping the max retries. May as well wait a bit longer between attempts.